screens/Updates.js

```diff
@@ -1,21 +1,28 @@
 import React from "react";
-import { StyleSheet, StatusBar, View } from "react-native";
+import { View, ScrollView } from "react-native";
 import { Appbar } from "react-native-paper";
+import Constants from "expo-constants";
+import { DEFAULT_APPBAR_HEIGHT } from "../node_modules/react-native-paper/src/components/Appbar/Appbar.tsx";
```

Comment:
This looks like a bad idea tbh.

Reply:
So the ScrollView doesn't let me scroll to the last post unless I pad the bottom of LatestUpdates with [header height + our 16 padding for ScrollView content]. I couldn't find a way to get header height in react-native-paper so I manually calculated it.

In hindsight, these values appear to be constants so this calculation could be avoided in favor of a magic number -> which again doesn't seem like a good idea.

I will try to find another solution.